src/generic/LaurentPoly.jl

```diff
@@ -417,7 +419,8 @@ end
 ################################################################################
 
 function map_coefficients(f, p::LaurentPolyWrap)
```

Comment:
I think this also should get a `cached` keyword, similar to the method for polynomials.